actions/st2_version.yaml

```diff
@@ -5,12 +5,12 @@
   description: "Get the latest stable or unstable version of St2"
   enabled: true
   parameters:
-    version:
+    branch:
       type: "string"
       required: true
       enum:
         - stable
-        - unstable
+        - latest
```

Comment:
Replaced keywords here: `unstable` with `latest` for consistency with [st2_deploy](https://github.com/StackStorm/st2/blob/4dc09d56d0dc1295f11e148b0be10e572c68d10a/tools/st2_deploy.sh#L99) /cc @dzimine 

But maybe this format would be better?
```sh
!st2 version dev
```

Especially when:
```sh
# st2 --version
st2 ('0.13dev.156',)
```

Don't know.